service/framework.go

```diff
@@ -384,6 +400,18 @@ func newPodFramework(config Config) (*framework.Framework, error) {
 	}
 
 	initCtxFunc := func(ctx context.Context, obj interface{}) (context.Context, error) {
+		{
+			container := loggercontext.NewContainer()
+
+			pod, err := key.ToPod(obj)
+			if err != nil {
+				return nil, microerror.Mask(err)
+			}
+			container.KeyVals["cluster"] = key.ClusterIDFromPod(pod)
+
+			ctx = loggercontext.NewContext(ctx, container)
+		}
+
```

Comment:
Here is the context prepared and the cluster ID is set based on the pod. Every log issuance using the context can now omit additional garbage and the logs become more consistent.